### ISSUE TITLE
508-defect-2: Addresses of mental health facilities are missing on make an appointment

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21477]
Label: 20133

    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

Issue Title: Make an appointment, addresses of mental health facilities missing
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a person with a disability who uses public transportation, it is essential that I have the exact address of my destination when booking rideshare or para-transit.
<h2>Details</h2>
Under "Walk-in mental health appointments" and "Walk-in drug and alcohol appointments" the address for the facility is missing.

Acceptance Criteria

Environment

Steps to Recreate

<h2>Proposed Solution (if known)</h2>
Add the address above the link to the map.

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs
